docs/providers/aws/guide/functions.md

```diff
@@ -463,15 +463,15 @@ functions:
 
 By default, the framework creates function versions for every deploy. This behavior is optional, and can be turned off in cases where you don't invoke past versions by their qualifier. If you would like to do this, you can invoke your functions as `arn:aws:lambda:....:function/myFunc:3` to invoke version 3 for example.
 
+Versions are not cleaned up by serverless, so make sure you use a plugin or other tool to prune sufficiently old versions. The framework can't clean up versions because it doesn't have information about whether older versions are invoked or not. This feature adds to the number of total stack outputs and resources because a function version is a separate resource from the function it refers to.
+
 To turn off this feature, set the provider-level option `versionFunctions`.
```

Comment:
To be honest, now this line seems a little out of context - maybe we should rephrase it as `To turn off function versioning, ...`. What do you think?

Reply:
I agree, that definitely seems more clear. I'll adjust it.